File: uri/salario/salario.cpp
#include <iostream>
#include <iomanip>

using namespace std;

typedef struct salario{
    int num;
    int horastrabalhadas;
    double paghora;
} Salario;

class URI{

    private:
        double calculasalario( int horas, double paghoras ){
            return ( horas * paghoras);
        }
    public:
        void show(){

            Salario func;
            cin >> func.num;
            cin >> func.horastrabalhadas;
            cin >> func.paghora;
            cout << "NUMBER = " << func.num << "\n";
            cout << "SALARY = U$ " << fixed << setprecision(2) << calculasalario(func.horastrabalhadas, func.paghora) << "\n";
        }
};

int main(void){
    URI pool;
    pool.show();
    return 0;
}

File: uri/salario/salario.go
package main

import (
	"C"
	"fmt"
)

type Salario struct {
	num, horas int32
	sal        float64
}

func calculasalario(horas int32, sal float64) float64 {
	return (float64(horas) * sal)
}

func main() {
	var foo Salario
	fmt.Scanf("%d\n", &foo.num)
	fmt.Scanf("%d\n", &foo.horas)
	fmt.Scanf("%g\n", &foo.sal)

	fmt.Printf("NUMBER = %d\nSALARY = U$ %.2f\n", foo.num, calculasalario(foo.horas, foo.sal))
}
